src/components/StarRating/starRating.test.js

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+import renderer from "react-test-renderer";
+
+import { StarRating } from "./index";
+
+it("displays the Restaurants Star rating correctly", () => {
+  const tree = renderer.create(<StarRating rating={2.5} />).toJSON();
+  expect(tree).toMatchSnapshot();
```

Comment:
Probably a bit much to have all rates, but maybe worth adding zero and a full-number.